### PR TITLE
Styleguide: fix image links, minor changes

### PR DESCRIPTION
**Pull request by dirkolbrich**
- fix links to ui images
- fix some spelling errors
- remove double whitespace
- sanitize links to resource books

### CODE REVIEW

File: stylebook/index.md [hunk 1]
```diff
@@ -9,43 +9,42 @@ layout: default
...
-
-In the long term, the book should settle down and become primarily a resource for developers and designers to refer to.  Most links should be removed.
+In the short term, this book surely will undergo massive changes and reorganization. It will also serve as an important part of a larger discussion within the FreeCAD community. Feel free to link to existing issues and pull requests, and projects.
 
+In the long term, the book should settle down and become primarily a resource for developers and designers to refer to. Most links should be removed.
 
 ## Parts of the UI
 
-![Parts of the UI]()
+![The FreeCAD UI and it's elements](ui.png)
```
Comment (chennes) [inline stylebook/index.md:20]:
```suggestion
![The FreeCAD UI and its elements](ui.png)
```

File: stylebook/index.md [hunk 2]
```diff
@@ -63,21 +62,21 @@ The workbench switcher is an important control to general FreeCAD workflow.  It
...
-5. Group locally similar tools into a toolgroup
-6. All tools in a toolbar should have a translatable and useful tooltip
-7. The toolbar should have a translatable and userful tooltip
-8. A toolbar should contain a minimum number of tools, ideally less than seven. Longer toolbars provide fewer options for customization and often get shortened or clipped offscreen.  They also increase cognitive burden since the user has to process the group as a whole.
+2. Too many toolbars create visual clutter. All tools compete for the users attention simultaneously. They are not context aware.
+3. Toolbars group related commands.
+4. Anticipate the user might disable the toolbar.
+5. Group locally similar tools into a toolgroup.
+6. All tools in a toolbar should have a translatable and useful tooltip.
+7. The toolbar itself should have a translatable and userful tooltip.
```
Comment (chennes) [inline stylebook/index.md:70]: As long as you're touching the line anyway...
```suggestion
7. The toolbar itself should have a translatable and useful tooltip.
```

**Review by chennes — APPROVED** [review]
I dropped a couple of comments on here: if you'd rather merge the original and do these in a separate pass (either yours or mine) let me know.

File: stylebook/index.md (hunk 1 above)
Comment (dirkolbrich) [reply stylebook/index.md:20]: Thanks. Will change. Its the little details

File: stylebook/index.md (hunk 2 above)
Comment (dirkolbrich) [reply stylebook/index.md:70]: Ha, good one. Have missed this.

**Comment by dirkolbrich** [comment]
Comments accepted and integrated.

**Pull request merged**